package.json

```diff
@@ -30,7 +30,8 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "^2.8.6",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "vitepress": "^1.2.3"
```

Comment:
do we have to add vitepress here? I think this dependency takes part already from docs folders, right?

Reply:
It does! Just remove it.